node/sc/bridge_manager_test.go

```diff
@@ -1988,7 +1989,7 @@ func TestDecodingLegacyAnchoringTx(t *testing.T) {
 }
 
 // DeployBridgeTest is a test-only function which deploys a bridge contract with some amount of KLAY.
-func (bm *BridgeManager) DeployBridgeTest(backend *backends.SimulatedBackend, local bool) (common.Address, error) {
+func (bm *BridgeManager) DeployBridgeTest(backend *backends.SimulatedBackend, amountOfBridgeContract int64, local bool) (common.Address, error) {
```

Comment:
`amountOfBridgeContract` is for depositing KLAY to the bridge contract, right?

What about...? I don't know what is good...
- KLAYinPeb
- depositKLAYinPeb
- value

Reply:
Yes. The list-up seems too general. I think it would be more understandable if the variable name contains the semantic of that value. So, how about `amountOfDeposit`?

Reply:
@hyunsooda Okay. I'm good with `amountOfDeposit`.

Reply:
Renamed.